v3/internal/templates/vanilla-ts/frontend/public/style.css

```
@@ -13,15 +13,15 @@
     -webkit-text-size-adjust: 100%;
-}
-
-* {
     user-select: none;
     -webkit-user-select: none;
     -moz-user-select: none;
     -ms-user-select: none;
 }
 
+* {
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)